.bazelrc

```diff
@@ -1,3 +1,9 @@
+common --enable_platform_specific_config
```

Comment:
We set different build flags per-host. 
This is because OR-tools compilation requires C++20, and the flag to set this is different for windows MSVC compilers and non-MSVC compilers.

Note that this flag is another barrier to cross-compilation, if that ever comes back onto the table.

Reply:
This command basically is a note to bazel that it should should select platform-specific configurations according to the host platform